src/Altinn.Platform/Altinn.Platform.PDF/src/main/java/altinn/platform/pdf/utils/TextUtils.java

```diff
@@ -196,4 +199,41 @@ public static String getInstanceGuid(String instanceId) {
 
     return instanceId.split("/")[1];
   }
+
+  /**
+   * Gets the a language value by key. If key is not found the key is returned.
+   * @param key the key
+   * @param languageCode the language to fetch from
+   */
+  public static String getLanguageByKey(String key, String languageCode) {
```

Comment:
the function name is a bit misleading.. 
As we are getting a text resource rather than a language. Could the name be changed?

Reply:
might also be that the name of the map is what is confusing me..

Reply:
Agreed, renamed.